pinot-core/src/main/java/org/apache/pinot/core/query/reduce/BrokerReduceService.java

```diff
@@ -212,7 +249,9 @@ public BrokerResponseNative reduceOnDataTable(BrokerRequest brokerRequest,
           .addMeteredTableValue(rawTableName, BrokerMeter.ENTRIES_SCANNED_IN_FILTER, numEntriesScannedInFilter);
       brokerMetrics
           .addMeteredTableValue(rawTableName, BrokerMeter.ENTRIES_SCANNED_POST_FILTER, numEntriesScannedPostFilter);
-
+      brokerMetrics.addValueToTableGauge(rawTableName, BrokerGauge.OFFLINE_THREAD_CPU_TIME_NS, offlineThreadCpuTimeNs);
```

Comment:
Further down around line 443, we are also logging few metrics. Please add these 2 stats as well to the logger line

Reply:
Sorry, I meant to log this in BaseBrokerRequestHandler code by calling the corresponding getters on BrokerResponseNative

Reply:
done